benchmarks/dynamo/common.py

```diff
@@ -2057,17 +2084,38 @@ def deepcopy_and_maybe_ddp(model):
                     # TB and TIMM use list example_inputs
                     # HF use dict example_inputs
                     if isinstance(example_inputs, dict):
-                        raise RuntimeError(
-                            "expect example_inputs as list/tuple, but got dict. need to support kwargs in torch._export.export"
+                        example_kwargs = example_inputs
```

Comment:
It seems there are quite some overlapping between the 1113-1134 with 2087-2109. Can we write a function to do that?

Reply:
oh wait i realized there already exists it 😂